.github/workflows/nodejs.yml

```diff
@@ -52,6 +52,7 @@ jobs:
           SENTRY_DSN: "${{ github.ref == 'refs/heads/releases' && secrets.SENTRY_DSN || '' }}"
           SOURCE_MAPS_UPLOAD: "${{ github.ref == 'refs/heads/releases' && secrets.SENTRY_UPLOAD_SOURCE_MAPS_URL || '' }}"
           SENTRY_AUTH: "${{ github.ref == 'refs/heads/releases' && secrets.SENTRY_AUTH || '' }}"
+          CF_API_KEY: "${{ secrets.CF_API_KEY }}"
```

Comment:
I am not sure if its right that it's called `CF_API_KEY` in the secrets and `REACT_CF_API_KEY` in the env plugin

Reply:
It's not working, I'm still working on it, I'm trying to pass the key from the actions to the code

Reply:
I doubt it will work in this repo because Davide has to set the secret for it here. In your repo it may do it.

Reply:
There's also a thing where the actions aren't exposed in PRs for security reasons, so it will always fail on the pr's action run

Reply:
Davide already set the key

Reply:
> There's also a thing where the actions aren't exposed in PRs for security reasons, so it will always fail on the pr's action run

I will keep it in mind, probably it's not working for that, still testing